spec/lib/requirements/backdate_checker_spec.rb

```diff
@@ -9,7 +9,7 @@
     end
 
     it "returns an issue if the date is in the future" do
-      date = Time.current.change(day: 1, month: 1, year: 2020)
+      date = Time.current.change(day: 1, month: 1, year: 1.year.from_now)
```

Comment:
This might be slightly weird as `1.year.from_now` returns an instance of `ActiveSupport::TimeWithZone` rather than an int. 

You can change this whole line to be date = `date = 1.year.from_now`

Reply:
Whoops. Yep, agreed.